styleguide/derek/global-components/tracking/tracking.js

```diff
@@ -181,6 +181,35 @@ Zoolander.Tracking = (function Tracking($) {
         eventNonInteraction: 0,
       });
     });
+
+    $('form.rsFilter-form').on('submit', (e) => {
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Resource Center',
+        eventAction: 'Resource Center Filter - Product',
+        eventLabel: $(e.target).find('option:selected').text(),
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Resource Center',
+        eventAction: 'Resource Center Filter - Type',
+        eventLabel: getCheckboxes($(e.target).find('input:checked')),
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+  }
```

Comment:
I'm not seeing the event firing here: https://rax.io/rsweb-9097-preview

Is it working for you locally? Additionally, since we're starting to use zoolander to manage data tracking, does it make sense to include GTM inside zoolander itself? That would make it a bit easier to test this kind of stuff.